package.json

```diff
@@ -130,7 +130,7 @@
 		"@babel/polyfill": "^7.12.1",
 		"@babel/preset-env": "^7.14.2",
 		"@babel/preset-react": "^7.12.7",
-		"@salesforce-ux/design-system": "^2.20.1",
```

Comment:
Have you check to see if anything is broken with the color contrast update by running the storybook?